nixos/modules/system/activation/top-level.nix

```diff
@@ -70,6 +70,7 @@ let
       ln -s ${config.system.build.etc}/etc $out/etc
       ln -s ${config.system.path} $out/sw
       ln -s "$systemd" $out/systemd
+      ln -s ${config.system.build.users-groups-spec} $out/users-groups.json
```

Comment:
Maybe we should stop cluttering $out

Reply:
> Maybe we should stop cluttering $out

Where would you be comfortable seeing this file?

Reply:
What about `$out/sw/share/nixos/useres-groups.json`? That way it would be neatly packed in a place where I'd expect nixos-specific data. Even feels a bit FHS-ish ;)
`$out/etc` would be another alternative but I don't think it fits there entirely